src/test/java/seedu/address/logic/commands/UpdatePhotoCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.AddressBookParser;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.ModelManager;
+
+public class UpdatePhotoCommandTest {
+
+    @Test
+    public void invalidIndex_exceptionThrown() {
```

Comment:
Consider renaming this to follow the [Java coding standard for test methods](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=featureUnderTest_testScenario_expectedBehavior())

Suggestion: `execute_invalidIndex_exceptionThrown`